app/models/enterprise.rb

```diff
@@ -199,7 +189,7 @@ def self.distinct_count
   end
 
   def activated?
-    confirmed_at.present? && sells != 'unspecified'
+    contact.confirmed? && sells != 'unspecified'
```

Comment:
I was working from this discussion about replacing email with a "contact" user that defaults to the owner but can be changed: https://community.openfoodnetwork.org/t/confirmation-emails-registration-flow/237